lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -216,11 +216,24 @@ namespace :foreman_tasks do
         end
       end
 
+      def self.count_duration(task)
+        duration = nil
+        if task.started_at && task.ended_at
+          duration = (task.ended_at - task.started_at).floor
+        elsif task.started_at && task.ended_at.nil?
+          duration = (Time.now - task.started_at).floor
+        else
+          duration = "N/A"
+        end
+        return duration
```

Comment:
In ruby, everything is an expression. A function implicitly returns the result of evaluating last expression in the function, explicit returns are only needed if you need to exit early. The same goes for `if`, result of evaluating an `if` statement is whatever the selected branch evaluated to. With this in mind, you should be able to reduce the code a bit